torch/csrc/jit/codegen/cuda/index_compute.h

```diff
@@ -207,6 +207,10 @@ class RootPredicateInfo {
     return root_ids_;
   }
 
+  const auto& consumerIds() const {
```

Comment:
I don't see any uses of this, where is it being used?

Reply:
Good catch. Likely code removed recently in devel branch. Let me clean it up.